pkg/packaging/target.go

```diff
@@ -70,6 +85,13 @@ func (t *Target) Parse(s string) error {
 		return err
 	}
 
+	// For now, set the default arch according to the given platform
+	if defaultArch, ok := defaultArchMap[t.Platform]; ok {
+		t.Arch = defaultArch
+	} else {
+		return fmt.Errorf("cannot select default arch for unknown platform %s", t.Platform)
+	}
+
```

Comment:
prefer happy path on left, reduce nesting

```suggestion
	// For now, set the default arch according to the given platform
	if defaultArch, ok := defaultArchMap[t.Platform]; !ok {
		return fmt.Errorf("cannot select default arch for unknown platform %s", t.Platform)
	}
	t.Arch = defaultArch
```

Reply:
Updated!